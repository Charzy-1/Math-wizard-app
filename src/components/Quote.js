import React from 'react';

const Quote = () => (
  <div className="Quote-container">
    <p>
      Mathematics is not all about numbers, equations, computations, or algorithms:
      It is about understanding -
      <strong>Williams Paul Thurston</strong>
    </p>
  </div>
);

export default Quote;
