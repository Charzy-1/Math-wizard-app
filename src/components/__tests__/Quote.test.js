import React from 'react';
import { render } from '@testing-library/react';
import Quote from '../Quote';

// Jest snapshot test
test('Quote component matches the snapshot', () => {
  const { asFragment } = render(<Quote />);
  expect(asFragment()).toMatchSnapshot();
});
