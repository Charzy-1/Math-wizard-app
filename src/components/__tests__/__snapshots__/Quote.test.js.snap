// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Quote component matches the snapshot 1`] = `
<DocumentFragment>
  <div
    class="Quote-container"
  >
    <p>
      Mathematics is not all about numbers, equations, computations, or algorithms: It is about understanding -
      <strong>
        Williams Paul Thurston
      </strong>
    </p>
  </div>
</DocumentFragment>
`;
